src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList.DuplicateTagException;
+
+public class Tagging {
+   private int status;
+   private Person person;
+   private UniqueTagList alltags;
+   
+   public Tagging(){
+	   
+   }
+   
+	public Tagging(int status, ReadOnlyPerson targetPerson, Tag tag) throws DuplicateTagException{
+		this.status=status;
+		if(targetPerson instanceof Person){
+			person=(Person)targetPerson;
+		}
+		
+		alltags.add(tag);
+	}
+	
+	public int getStatus(){
+		return status;
+	}
+	
+	public String toString(){
+		String action = (status==1) ? "+" : "-";
+		String result = action + " " + person.getName().toString() + " ";
+		for(Tag tag: alltags){
+			result += tag.toString();
+		}
+		return result;
```

Comment:
Coding standard violation! One more more of these problems detected:
1. extra/missing spaces.
2. brace style problem.
3. missing braces.
4. name doesn't follow naming convention